content/en/docs/aws/rds.md

```diff
@@ -0,0 +1,92 @@
++++
+title = "Amazon RDS"
+date = 2020-04-29T12:54:30-04:00
+description = "Using Amazon RDS for storing pipelines and metadata"
+weight = 98
++++
+
+This guide describes how to use Amazon RDS as your pipelines and metadata store.
+
+## Amazon Relational Database Service (Amazon RDS)
+
+Amazon RDS is a managed service that makes it easier to set up, operate, and scale a relational database in the AWS Cloud. It provides cost-efficient, resizable capacity for an industry-standard relational database and manages common database administration tasks. It has support for several engines such as  MySQL, MariaDB, PostgreSQL, Oracle, and Microsoft SQL Server DB engines.  
+
+### Deploy Amazon RDS MySQL in your environment
+
+Before deploying MySQL database using Amazon RDS, let's get configuration parameters that are needed such as VpcId, SubnetIds and SecurityGroupId
+
+```shell
+# Use these commands to find VpcId, SubnetId and SecurityGroupId if you deployed your EKS cluster using eksctl
+export AWS_CLUSTER_NAME=<YOUR EKS CLUSTER NAME>
+
+# Below command will retrieve your VpcId
+aws ec2 describe-vpcs --filters Name=tag:alpha.eksctl.io/cluster-name,Values=$AWS_CLUSTER_NAME | jq -r '.Vpcs[].VpcId'
```

Comment:
Not all AWS customers are using EKS, can we leave a comment here for non-eks users?

Reply:
Added comment for non-eks customers